lib/bankaccount.rb

```diff
@@ -0,0 +1,9 @@
+module BankAccount
+  require 'pathname'
+
+  ROOT_DIR = Pathname.new(__dir__).parent
+  require ROOT_DIR.join('lib/bankaccount/account')
+  require ROOT_DIR.join('lib/bankaccount/savingsaccount')
+  require ROOT_DIR.join('lib/bankaccount/checkingaccount')
+
+end
```

Comment:
Good to know. Is there a good time to actually use Pathname? (If it's something beyond my current level, I understand.)